### DISCUSSION TITLE
[Q&A] 代理问题 极空间部署docker

### DISCUSSION THREAD

**Discussion by FrankFuShMomentLab** [lobehub/lobehub#855] [General Question] [answered]
### 🧐 问题描述 | Proposed Solution

 使用Docker部署教程中的代理地址 ，OPENAI_PROXY_URL: https://api-proxy.com/v1，故障报错：

{
  "endpoint": "https://api-proxy.com/v1",
  "error": {
    "cause": {
      "errno": -3007,
      "code": "ENOTFOUND",
      "syscall": "getaddrinfo",
      "hostname": "api-proxy.com"
    }
  }
}”
“”
<img width="773" alt="1" src="https://github.com/lobehub/lobe-chat/assets/130262399/33334a0b-2f0d-4886-b920-8cf951ae0468">

<img width="481" alt="2" src="https://github.com/lobehub/lobe-chat/assets/130262399/d09e97b6-a3a0-4e44-91ea-08b3580ae3c4">

使用其它的代理地址，有别的报错

[a comment by an automated account is not shown]

**Comment by FrankFuShMomentLab** [comment]
<img width="809" alt="3" src="https://github.com/lobehub/lobe-chat/assets/130262399/da8ebbea-5a19-43e3-ac2a-45441df75aa0">

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
proxy 当然得用你自己的了…如果没有的话就不用填

[a comment by an automated account is not shown]
